package.json

```diff
@@ -0,0 +1,49 @@
+{
+  "name": "maintenance-tracker",
+  "version": "1.0.0",
+  "description": "Helps to track maintenance requests of user",
+  "main": "index.js",
+  "scripts": {
+    "build": "babel lib -d dist",
+    "pretest": "eslint --ignore-path .gitignore .",
+    "serve": "node dist/index.js",
+    "start": "nodemon lib/index.js --exec babel-node",
```

Comment:
https://codeburst.io/dont-use-nodemon-there-are-better-ways-fc016b50b45e Look for a nodemon alternative

Reply:
Ok will check it out and refactor my code